docs/core/dependency-loading/collect-details.md

```diff
@@ -0,0 +1,164 @@
+| AssemblyLoader/Start | MyLibrary, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null | //1/2/1/   |         |                                                                           |
+| AssemblyLoader/Stop  | MyLibrary, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null | //1/2/1/   | True    | C:\samples\AssemblyLoading\AssemblyLoading\bin\Debug\net5.0\MyLibrary.dll |
+| AssemblyLoader/Stop  |                  MyLibrary, Culture=neutral, PublicKeyToken=null | //1/2/     | True    | C:\samples\AssemblyLoading\AssemblyLoading\bin\Debug\net5.0\MyLibrary.dll |
+
+We can then look at the `ResolutionAttempted` events with the activity ID from the outer load to determine the step at which the assembly was successfully resolved. This time, the events will show that the `AssemblyLoadContextResolvingEvent` stage was successful. The `ResultAssemblyPath` field shows the path of the resulting assembly.
+
+![PerfView successful ResolutionAttempted events image](media/collect-details/resolution-attempted-success.png)
+
+| Event Name                         |                                    AssemblyName | Stage                             | Result           | ResultAssemblyPath |
+|------------------------------------|-------------------------------------------------|-----------------------------------|------------------| ------------------ |
+| AssemblyLoader/ResolutionAttempted | MyLibrary, Culture=neutral, PublicKeyToken=null | FindInLoadContext                 | AssemblyNotFound |                    |
+| AssemblyLoader/ResolutionAttempted | MyLibrary, Culture=neutral, PublicKeyToken=null | ApplicationAssemblies             | AssemblyNotFound |                    |
+| AssemblyLoader/ResolutionAttempted | MyLibrary, Culture=neutral, PublicKeyToken=null | AssemblyLoadContextResolvingEvent | Success          | C:\samples\core\extensions\AssemblyLoading\AssemblyLoading\bin\Debug\net5.0\MyLibrary.dll |
+
+Looking at `AssemblyLoadContextResolvingHandlerInvoked` events will show that the handler named `OnAssemblyLoadContextResolving` was invoked. The `ResultAssemblyPath` field shows the path of the assembly returned by the handler.
+
+![PerfView successful extension point events image](media/collect-details/extension-point-success.png)
+
+| Event Name                                                | AssemblyName                                    | HandlerName                    | ResultAssemblyPath |
+|-----------------------------------------------------------|-------------------------------------------------|--------------------------------|--------------------|
+| AssemblyLoader/AssemblyLoadContextResolvingHandlerInvoked | MyLibrary, Culture=neutral, PublicKeyToken=null | OnAssemblyLoadContextResolving | C:\samples\core\extensions\AssemblyLoading\AssemblyLoading\bin\Debug\net5.0\MyLibrary.dll |
```

Comment:
There are a few references to file/folder paths in the articles. File paths should be enclosed in _italics_